src/index.js

```diff
@@ -166,6 +182,16 @@ const App = () => {
         </a>
       </div>`
 
+  const renderDeletePlaylistButton = () => 
+      playList.length > 0 && html`
+      <div
+        class=${css.outlineButton}
```

Comment:
Can you use the default danger button of [tailwind-elements](https://tailwind-elements.com/docs/standard/components/buttons/)?
![image](https://user-images.githubusercontent.com/30677819/170368178-0b2bd581-c5c3-4bd1-a8dd-91a07c16df7a.png)